pandas/tests/series/test_constructors.py

```diff
@@ -804,7 +804,7 @@ def test_constructor_floating_data_int_dtype(self, frame_or_series):
         # Long-standing behavior (for Series, new in 2.0 for DataFrame)
         #  has been to ignore the dtype on these;
         #  not clear if this is what we want long-term
-        expected = frame_or_series(arr)
+        # expected = frame_or_series(arr)
```

Comment:
Are these supposed to be commented out?

Reply:
Yes, they were unused, but the comment above is a bit confusing, that’s why I did not delete them
Outright